CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing
+**Table of Contents**
+- [Getting Started](#getting-started) (1st draft)
```

Comment:
"Getting Started" vs "Setup and Run" is confusing to me... is this really more of "Welcome"?  Maybe not, but right now the distinction isn't clear. :-)

Reply:
"Welcome" sounds right.